clio-server/src/main/scala/org/broadinstitute/clio/server/dataaccess/elasticsearch/DocumentReadGroup.scala

```diff
@@ -2,15 +2,16 @@ package org.broadinstitute.clio.server.dataaccess.elasticsearch
 
 import java.time.OffsetDateTime
 
-case class DocumentReadGroup(analysisType: Option[String],
+case class DocumentReadGroup(flowcellBarcode: String,
+                             lane: Int,
+                             libraryName: String,
+                             location: String,
```

Comment:
Maybe overkill, but I'd think of `location` (here and elsewhere) as more of an enum than a string. Scala's actual enum type doesn't have a very good reputation but you can simulate what you need with `sealed trait`s and `case object`s.